dotCMS/src/main/java/com/dotcms/keyvalue/business/KeyValueAPI.java

```diff
@@ -163,4 +163,20 @@ public default KeyValue fromContentlet(final Contentlet contentlet) {
     public KeyValue get(final String key, final long languageId, final ContentType contentType, final User user,
                     final boolean respectFrontendRoles);
 
+    /**
+     *
+     * Returns a list of {@link KeyValue} that the key starts with the specified key, language and content type.
```

Comment:
javadoc: *Returns a list of {@link KeyValue} **whose keys start with the provided key**